src/python/CRABClient/Commands/getlog.py

```diff
@@ -25,41 +21,23 @@ class getlog(getcommand):
 
     def __call__(self):
         if self.options.short:
-            inputlist = {'subresource': 'webdir', 'workflow': self.cachedinfo['RequestName']}
+            taskname = self.cachedinfo['RequestName']
+            inputlist = {'subresource': 'webdir', 'workflow': taskname}
             serverFactory = CRABClient.Emulator.getEmulator('rest')
-            server = serverFactory(self.serverurl, self.proxyfilename, self.proxyfilename, version=__version__) 
+            server = serverFactory(self.serverurl, self.proxyfilename, self.proxyfilename, version=__version__)
             uri = self.getUrl(self.instance, resource = 'task')
-            dictresult, status, reason =  server.get(uri, data = inputlist)
-            self.logger.info('Server result: %s' % dictresult['result'][0])
-            dictresult = self.processServerResult(dictresult)
-            if status != 200:
-                msg = "Problem retrieving information from the server:\ninput:%s\noutput:%s\nreason:%s" % (str(inputlist), str(dictresult), str(reason))
-                raise RESTCommunicationException(msg)
+            webdir = getProxiedWebDir(taskname, self.serverurl, uri, self.proxyfilename, self.logger.debug)
+            if not webdir:
+                dictresult, status, reason =  server.get(uri, data = inputlist)
+                webdir = dictresult['result'][0]
+                self.logger.info('Server result: %s' % webdir)
+                dictresult = self.processServerResult(webdir)
```

Comment:
processServerResult does nothing, but in principle I think one would use it as:
```
if not webdir:
    inputlist = {'subresource': 'webdir', 'workflow': taskname}
    dictresult, status, reason =  server.get(uri, data = inputlist)
    self.logger.info('Server result: %s' % dictresult)
    dictresult = self.processServerResult(dictresult)
    webdir = dictresult['result'][0]
```
Or you may prefer to remove processServerResult (defined in getcommand) at all.